common/src/main/java/com/wynntils/overlays/StatusEffectsOverlay.java

```diff
@@ -108,4 +133,38 @@ private void updateTextRenderSetting() {
     protected TextRenderSetting getTextRenderSetting() {
         return textRenderSetting;
     }
+
+    private Stream<StatusEffectWithProperties> stackEffects(List<StatusEffect> effects) {
+        LinkedHashMap<String, StatusEffectWithProperties> effectsToRender = new LinkedHashMap<>();
```

Comment:
```suggestion
        Map<String, StatusEffectWithProperties> effectsToRender = new LinkedHashMap<>();
```

Should be good to lower this type?

Reply:
Doesn't impact the usage, done